panels/dmx.py

```diff
@@ -105,7 +105,7 @@ def draw(self, context):
         #layout.menu("dmx.menu.universe", text="...", icon="FILE_VOLUME")
 
 class DMX_PT_DMX_ArtNet(Panel):
-    bl_label = "ArtNet"
+    bl_label = "Ethernet"
```

Comment:
It makes a lot of sense moving on from "ArtNet" now that we're including sACN. However, "Ethernet" sounds a little too technical for me.
Do you think "Network" could be a better word?

Reply:
I like the Network word, it is definitively better. I renamed it already, if people vote for Ethernet (i would be surprised), we can rename it again...